src/frontend/web_application/src/scenes/Timeline/components/MessageItem/style.scss

```diff
@@ -45,63 +50,82 @@
   &--unread {
     position: relative;
     background-color: $co-color__fg__back;
-    color: $co-color__fg__text;
   }
 
   &--is-selected {
     background-color: $co-color__bg__back--higher;
   }
 
-  &__col-avatars {
+  &__col-avatar {
     @include flex-grid-column(shrink);
+    margin-left: $s-message-item__pi-width;
   }
 
   &__col-title {
     @include flex-grid-size;
     min-width: 0; // https://github.com/zurb/foundation-sites/pull/8511
   }
 
-  &__title-content {
+  &__title {
     @include flex-grid-row($size: expand);
+    align-items: center;
   }
 
-  &__title-author {
-    @include flex-grid-column(12);
-    font-weight: 600;
+  &__author { @include flex-grid-size(12); }
+
+  &__author-name {
     color: $co-color__primary--lower;
+    font-size: $co-font__size;
+    font-weight: 600;
   }
 
-  &__title-subject {
-    @include flex-grid-column(12);
+  &__topic {
+    @include flex-grid-size(12);
+
+    &--unread { font-weight: 600; }
   }
 
+  &__subject { color: $co-color__fg__text--lower; }
+  &__excerpt { color: $co-color__fg__text--high; }
+
   &__col-file {
     @include flex-grid-column(shrink);
     display: none;
   }
 
   &__col-dates {
-    @include flex-grid-size(2);
+    @include flex-grid-column(shrink);
+    color: $co-color__fg__text--high;
     text-align: right;
+
+    &--unread {
+      color: $co-color__fg__text--low;
+    }
   }
 
+  &__type-label { display: none; }
+
   &__col-select {
-    @include flex-grid-column(shrink);
     display: none;
+    @include flex-grid-column(shrink);
+    margin-left: auto;
   }
 
   @include breakpoint(medium) {
-    &__col-select,
-    &__col-file { display: block; }
+    padding-top: 0;
```

Comment:
is this due to foundation flex grid ?

may be it should be better to use vanilla flex directives ?

Reply:
not related to foundation grid: this is only related to UI (padding top and bottom only on small views)

Reply:
👍